app/assets/stylesheets/events.scss

```diff
@@ -54,6 +54,7 @@
     border: 1px solid #CCCCCC !important;
     border-radius: 0px !important;
     width: 90%;
+    color: #ffffff !important; 
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line contains trailing whitespace